plugins/tech-radar/README.md

```diff
@@ -91,8 +122,8 @@ export class MyOwnClient implements TechRadarApi {
     const data = await fetch('https://mydata.json').then(res => res.json());
 
     // maybe you'll need to do some data transformation here to make it look like TechRadarLoaderResponse
-
-    return data;
+    // Need a converter for the Date object
+    return convertTechRadarData(data);
```

Comment:
```suggestion
    // For example, this converts the timeline dates into date objets
    return {
      ...data,
      entries: data.entries.map(entry => ({
        ...entry,
        timeline: entry.timeline.map(timeline => ({
          ...timeline,
          date: new Date(timeline.date),
        }))
      }))
    }
```

Reply:
That's even a better solution :)
Updating PR.